pythonfmu/pythonfmu-export/cpp/PySlaveInstance.cpp

```diff
@@ -436,6 +441,7 @@ void PySlaveInstance::handle_py_exception(const std::string& what, PyGILState_ST
 
 PySlaveInstance::~PySlaveInstance()
 {
+    clearStrBuffer();
```

Comment:
We should move that call in `cleanPyObject` as it should be called when an error occurs too.